src/mito2/src/test_util.rs

```diff
@@ -558,6 +560,36 @@ impl Iterator for VecBatchReader {
     }
 }
 
+pub fn new_binary_batch_builder(
+    primary_key: &[u8],
+    timestamps: &[i64],
+    sequences: &[u64],
+    op_types: &[OpType],
+    field_column_id: ColumnId,
+    field: Vec<Vec<u8>>,
+) -> BatchBuilder {
+    let mut builder = BatchBuilder::new(primary_key.to_vec());
+    builder
+        .timestamps_array(Arc::new(TimestampMillisecondArray::from_iter_values(
+            timestamps.iter().copied(),
+        )))
+        .unwrap()
+        .sequences_array(Arc::new(UInt64Array::from_iter_values(
+            sequences.iter().copied(),
+        )))
+        .unwrap()
+        .op_types_array(Arc::new(UInt8Array::from_iter_values(
+            op_types.iter().map(|v| *v as u8),
+        )))
+        .unwrap()
+        .push_field_array(
+            field_column_id,
+            Arc::new(LargeBinaryArray::from_iter_values(field)),
+        )
+        .unwrap();
```

Comment:
Since only the `test_read_large_binary` test uses it, we don't need to define a method to construct a batch builder. Let's merge this into `new_batch_with_binary_by_range()`.